addons/cargo/functions/fnc_removeCargoItem.sqf

```diff
@@ -0,0 +1,48 @@
+/*
+ * Author: 654wak654
+ * Removes a cargo item from the vehicle.
+ *
+ * Arguments:
+ * 0: Item Classname <STRING>
+ * 1: Vehicle <OBJECT>
+ * 2: Amount <NUMBER> (default: 1)
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["item", vehicle] call ace_cargo_fnc_removeCargoItem
+ *
+ * Public: Yes
+ */
+#include "script_component.hpp"
+
+params ["_itemClass", "_vehicle", ["_amount", 1]];
+TRACE_3("params",_itemClass,_vehicle,_amount);
+
+private _loaded = _vehicle getVariable [QGVAR(loaded), []];
+
+private _itemsToRemove = [];
+private _addedSpace = 0;
+
+{
+    if (count _itemsToRemove == _amount) exitWith {};
+
+    if (_x == _itemClass || {typeOf _x == _itemClass}) then {
```

Comment:
This fails if objects are in the loaded array
```
   Error position: <== _itemClass || {typeOf _x == _itemClas>
   Error Generic error in expression
 File z\ace\addons\cargo\functions\fnc_removeCargoItem.sqf, line 15
```
Need to check type, then compare, something like
```
(((_x isEqualType "") && {_x == _itemClass}) || {(_x isEqualType objNull) && {typeOf _x == _itemClass}})
```